ui-ngx/src/app/modules/home/pages/rulechain/rule-node-config.component.ts

```diff
@@ -207,4 +207,15 @@ export class RuleNodeConfigComponent implements ControlValueAccessor, OnInit, On
       this.definedConfigComponent.validate();
     }
   }
+
+  private transformControls(configuration: RuleNodeConfiguration): any {
+    const result = {};
+    for (const [key, value] of Object.entries(configuration)) {
+      result[key] = {
+        value,
+        disabled: this.disabled
```

Comment:
What if some inner components are disabled based on the logic specific to the particular rule node implementation? This implementation will work with primitive forms. But will not work with complex forms.

Reply:
Ok, I will review and update pr